files/bin/deploy.py

```diff
@@ -200,13 +235,6 @@ def main() -> None:
         oci_credentials_eph = os.environ.get("OCI_CREDENTIALS_EPH")
         oci_config_eph = os.environ.get("OCI_CONFIG_EPH")
 
-        cred_params = [
```

Comment:
we can get rid of all references to the `aws_credentials_eph`, `gcp_credentials_eph`, `oci_credentials_eph`, and `oci_config_eph` variables.